### PR TITLE
Oracle8 build

### PR DESCRIPTION
**Pull request by JasonMarechal25**
Add new workflow to build on oracle 8 platform.

### CODE REVIEW

File: .github/workflows/build_oracle8.yml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        with:
+          prefix: "../rte-antares-deps-Release/"
+
+      - name: Compile TBB
+        uses: ./.github/workflows/compile-tbb
+
+      - name: Install dependencies
+        run: |
+          source /opt/rh/gcc-toolset-9/enable
+          pip3 install wheel
```
Comment (a-zakir) [inline .github/workflows/build_oracle8.yml]:
```suggestion
          
```
it's added also in requirements.txt. why do we need this package?

Comment (JasonMarechal25) [reply .github/workflows/build_oracle8.yml]: For some reason it is required (I think by pyinstaller) to be installed and can't be installed by requirements

**Review by a-zakir — COMMENTED** [review]
i'm ok with this. the branch has to be updated with develop before merge.

**Comment by JasonMarechal25** [comment]
Need to look into some CI failure

**Comment by JasonMarechal25** [comment]
Blocked by https://github.com/AntaresSimulatorTeam/Antares_Simulator/issues/1741

File: .github/workflows/build_oracle8.yml [hunk 2]
```diff
@@ -0,0 +1,147 @@
+name: Oracle 8 Build
+
+on:
+  merge_group:
+  push:
+    branches:
+      - main
```
Comment (a-zakir) [inline .github/workflows/build_oracle8.yml]:
```suggestion
```

File: .github/workflows/build_oracle8.yml [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+      XPRESS_CONTAINER: ${GITHUB_WORKSPACE}/xpress/bin
+      XPRS_LIB_Path_CONTAINER: ${GITHUB_WORKSPACE}/xpress/lib
+
+    steps:
+
+      - name: Install System
+        run: |
+          dnf install -y epel-release git cmake wget rpm-build redhat-lsb-core openmpi-devel
+          dnf install -y unzip libuuid-devel boost-test boost-devel gcc-toolset-9-toolchain zlib-devel python3-devel
+
```
Comment (a-zakir) [inline .github/workflows/build_oracle8.yml:67]: what's the purpose of python3-devel? later on line 86 you installed python3.

Comment (JasonMarechal25) [reply .github/workflows/build_oracle8.yml:67]: devel version is required to build wheel which is required to pyinstaller

**Review by a-zakir — APPROVED** [review]

**Comment by a-zakir** [comment]
re-run checks before merge

**Pull request merged**